prusti-specs/src/lib.rs

```diff
@@ -501,4 +515,4 @@ pub fn predicate(tokens: TokenStream) -> TokenStream {
         #[prusti::pred_spec_id_ref = #spec_id_str]
         #cleaned_fn
     }
-}
+}
```

Comment:
```suggestion
}

```

(and in general, please try to keep the line ends)